app/src/main/java/me/capcom/smsgateway/modules/localserver/routes/InboxRoutes.kt

```
@@ -0,0 +1,208 @@
+        post("refresh") {
+            if (!requireScope(AuthScopes.InboxRefresh)) return@post
+
+            val request = try {
+                call.receive<PostMessagesInboxExportRequest>().validate()
+            } catch (e: IllegalArgumentException) {
+                call.respond(
+                    HttpStatusCode.BadRequest,
+                    mapOf("message" to (e.message ?: "Invalid request"))
+                )
+                return@post
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)